pkgs/development/python-modules/llama-parse/default.nix

```diff
@@ -22,7 +23,10 @@ buildPythonPackage rec {
 
   build-system = [ poetry-core ];
 
-  dependencies = [ llama-index-core ];
+  dependencies = [
+    llama-cloud-services
+    llama-index-core
```

Comment:
```suggestion
```
Upstream doesn't list it as a dependency.

Reply:
Indeed, that dependency may have gotten there before some refactoring :thinking: 

Upstream (newer version than nixpkgs) https://github.com/run-llama/llama_cloud_services/blob/main/llama_parse/pyproject.toml

Reply:
Removed

Reply:
It's still there

Reply:
Now really gone.